apis/swagger.yml

```diff
@@ -1259,3 +1285,93 @@ definitions:
     description: The status of the container. For example, "running" or "exited".
     type: "string"
     enum: ["created", "running", "stopped","paused", "restarting", "removing", "exited", "dead"]
+  NetworkCreateRequest:
+    type: "object"
+    description: "contains the request for the remote API: POST /networks/create"
+    properties:
+      Name:
+        description: "Name is the name of the network."
+        type: "string"
+      NetworkCreate:
+        $ref: "#/definitions/NetworkCreate"
+  NetworkCreateResponse:
+    type: "object"
+    description: "contains the response for the remote API: POST /networks/create"
+    properties:
+      ID:
+        description: "ID is the id of the network."
+        type: "string"
+      Warning:
+        description: "Warning means the message of create network result."
+        type: "string"
+  NetworkCreate:
+      type: "object"
+      description: "is the expected body of the \"create network\" http request message"
+      properties:
+        CheckDuplicate:
+          type: "boolean"
+          description: "CheckDuplicate is used to check the network is duplicate or not."
+        Driver:
+          type: "string"
+          description: "Driver means the network's driver."
+        Scope:
+          type: "string"
+        EnableIPv6:
+          type: "boolean"
+        IPAM:
+          type: "object"
+          $ref: "#/definitions/IPAM"
+        Internal:
```

Comment:
Add descriptions for these fields, since maybe we are not familiar with them?

I am also wondering if we need to remove some, because we will never use them, such as `Ingress`, `Attachable`.

Reply:
OK, I will add descriptions.
Some parameters will be used in the future.